Dockerfile

```diff
@@ -41,8 +41,12 @@ RUN apt-get update && \
 
 # binary downloads
 ARG TFSEC_VERSION="v0.24.1"
+ARG PACKER_VERSION="v1.6.1"
 RUN curl -L https://github.com/liamg/tfsec/releases/download/${TFSEC_VERSION}/tfsec-linux-amd64 -o /usr/local/bin/tfsec && \
```

Comment:
Tried going to this URL. Got 404. Is this the updated path? https://github.com/liamg/tfsec/releases/tag/v0.24.1/

Reply:
Touched base with @ericSeiso, link is good as-is.  Resolving